OMS/DBDataAcesses/GetDataFormDB.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Data;
+using Configuration;
+using FileModel;
+namespace DBDataAcesses
+{
+    public class GetDataFromDB: BaseProcessor
+    {
+        public static List<WareHouseModel> GetAllWareHouses()
+        {
+            var wareHouses = new List<WareHouseModel>();
+            SqlConnection i = null;
```

Comment:
Variable names should be human understandable.
Please rename sql connection object name